modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/source.jsp

```diff
@@ -127,7 +127,7 @@ List<Map<String, Object>> classTypesList = new ArrayList<>();
 					<%
 					for (ClassType classType : classTypes) {
 						if (Arrays.binarySearch(assetSelectedClassTypeIds, classType.getClassTypeId()) < 0) {
```

Comment:
This looks like the only instance of binary search where we don't sort first. Is there a reason why sorting is not necessary, or is this a bug? If we need to sort this first, then my previous comment about switching the order of right and left lists would no longer be valid.